Lib/glyphsLib/builder/kerning.py

```diff
@@ -24,25 +24,34 @@ def to_ufo_kerning(self):
     for master in self.font.masters:
         master_id = master.id
         kerning_source = master.metricsSource.id  # Maybe be a linked master
-        if kerning_source in self.font.kerning:
-            kerning = self.font.kerning[kerning_source]
+        if kerning_source in self.font.kerningLTR:
+            kerning = self.font.kerningLTR[kerning_source]
             _to_ufo_kerning(self, self._sources[master_id].font, kerning)
+        if kerning_source in self.font.kerningRTL:
+            kerning = self.font.kerningRTL[kerning_source]
+            _to_ufo_kerning(self, self._sources[master_id].font, kerning, "RTL")
```

Comment:
since both kerningLTR and kerningRTL are merged into the same ufo.kerning dictionary, is it ever possible that a clash might occur? I.e. the same kern pair defined in both kerningLTR and kerningRTL (potentially with different kern values), which would then mean that the RTL coming last would overwrite the LTR kern pair with the same (first, second) key?
I suppose this is possible in theory given Glypsh3 kerning is now defined in two split dictionaries, so glyphsLib should either raise some error or issue a warning about the ambiguous kern pair.